src/fprime/fpp/visualize.py

```diff
@@ -126,6 +130,7 @@ def run_fprime_visualize(
     config = {"SOURCE_DIRS": [str(viz_cache.resolve())]}
     app = construct_app(config)
     try:
+        webbrowser.open(f"http://localhost:{parsed.gui_port}")
```

Comment:
Should this come after app.run() ? Not that any browser would load that quickly... but just because?

Reply:
No, because app.run will block until an interrupt.

Reply:
To guarantee ordering, we'd need to spin-off a thread.  I am not sure that is worth it.

Reply:
Ohh right right, sounds good